README.md

```diff
@@ -208,6 +208,26 @@ chmod +x purge
 This provides a simple and convenient way of building a sophisticated DevOps toolbox
 for your Linux shell.
 
+### Building extension
+
+Create a ThingWorx extension, as a zip file with proper `metadata.xml`.
+
+In `build` mode the folder containing the entities to put in the extension is the first 
+parameter, and the extensin name is the second. 
+filename is the second:
+A third optional parameter can be used to specify extension version. If omitted, default
```

Comment:
Let's simplify it by making it mandatory.

Reply:
Done.